src/logging.h

```diff
@@ -119,7 +119,7 @@ namespace BCLog {
         /** Returns whether logs will be written to any output */
         bool Enabled() const
         {
-            LockGuard scoped_lock(m_cs);
+            StdLockGuard scoped_lock(m_cs);
             return m_buffering || m_print_to_console || m_print_to_file;
```

Comment:
18635 needs another `LockGuard` -> `StdLockGuard` once you do my previous comment